Files/UserControls/NavigationToolbar.xaml

```diff
@@ -965,6 +968,135 @@
                         </Flyout>
                     </AppBarButton.Flyout>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="ArrangementOptions"
+                    x:Uid="NavToolbarArrangementOptionsButton"
+                    Width="36"
+                    Height="32"
+                    x:Load="{x:Bind IsPageTypeNotHome, Mode=OneWay}"
+                    AccessKey="S"
+                    AutomationProperties.Name="Arrangement options"
+                    Background="Transparent"
+                    Style="{StaticResource ToolBarButtonStyle}"
+                    ToolTipService.ToolTip="Arrangement options">
+                    <FontIcon FontSize="14" Glyph="&#xF168;" />
+                    <AppBarButton.Flyout>
+                        <Flyout>
```

Comment:
Should the flyout be its own page or user control so that it's easier to reuse in other places?